CONTRIBUTING.md

```diff
@@ -18,4 +18,16 @@ Type classes
  - Hierarchy
 	 - For each trait of the hierarchy, a particular typeclass must appear only once as return type in one hierarchy level (`BH1`, `BH2`, ...).
 	 - Companion objects should only declare implicit instances for the type classes at the top of a hierarchy. 
-
+ - Classes
+     - Each typeclass `Foo` should have the following traits defined:
+        - `FooClass`: The definition of the typeclass.
+        - `FooSyntax`: Syntax macros for the typeclass methods.
+        - `FooFunctions`: A trait containing the typeclass's methods, but with
+                          the typeclass as a type parameter.
```

Comment:
as an implicit parameter?

Reply:
Thanks. I originally had a confusing attempt to say "with the typeclass instance as a type parameter" or something but I trashed it as insufficiently clear.